sdk/communication/communication-chat/samples/typescript/src/participantsOperations.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to use the ChatThreadClient to do participant operations
+ */
+
+import { ChatClient } from "@azure/communication-chat";
+import { AzureCommunicationTokenCredential, getIdentifierKind } from "@azure/communication-common";
+import { CommunicationIdentityClient } from "@azure/communication-identity";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export const main = async () => {
+  const connectionString =
+    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
+
+  const identityClient = new CommunicationIdentityClient(connectionString);
+  const user = await identityClient.createUser();
+  const userToken = await identityClient.getToken(user, ["chat"]);
+  const userJhon = await identityClient.createUserAndToken(["chat"]);
```

Comment:
Not a technical thing, but maybe a more gender neutral name or a female name for this user? we're already using Jack later.

Reply:
I like that! Renamed to "Sue".